oca_dependencies.txt

```diff
@@ -0,0 +1,18 @@
+# List the OCA project dependencies, one per line
+# Add a repository url and branch if you need a forked version
+#
+# Examples
+# ========
+#
+# To depend on the standard version of sale-workflow, use:
+# sale-workflow
+#
+# To explicitely give the URL of a fork, and still use the version specified in
+# .travis.yml, use:
+# sale-workflow https://github.com/OCA/sale-workflow
+#
+# To provide both the URL and a branch, use:
+# sale-workflow https://github.com/OCA/sale-workflow branchname
+
+account_asset_management
+account-financial-tools
```

Comment:
You just need the line with the repo name `account-financial-tools`.
Remove the line with module name `account_asset_management` - that's the reason the CI build is failing.